package.json

```diff
@@ -18,30 +18,31 @@
     "@openfeature/web-sdk": "^0.3.1-experimental",
     "@opentelemetry/api": "^1.3.0",
     "@protobuf-ts/grpc-transport": "2.9.0",
+    "@protobuf-ts/runtime-rpc": "^2.9.0",
```

Comment:
This dependency was simply missing, it should have been added with a recent deprecation to protobuf.